### ISSUE TITLE
Does this extension stop buffering also?

### ISSUE THREAD

**Issue by roberto2525** [Eloston/disable-html5-autoplay#9]
I would like to know if this extension prevents HTML5 videos from both buffering and auto-playing? Or does it only stop them from auto-playing?

The reason I ask is that I have a limited 100MB daily bandwidth plan using my 3 USB key, and so I want to conserve bandwidth while browsing. 

Thank you.

**Comment by Eloston** [comment]
Right now, it only stops autoplaying. It is a pretty low priority issue since it isn't part of the main functionality of this extension.

However if enough people want this feature, I will increase the priority.

**Comment by SeaWolfArch** [comment]
I would love to see that feature implemented, it would make this extension more useful :)

**Comment by roberto2525** [comment]
I agree with Seawolfarch, the extension would be better if it also stops buffering.

Eloston, if you add this enhancement, please reply to this post so I will add the extension to my browser. I need it to also stop buffering. Thank you.

**Comment by himanshu0** [comment]
+1 This is is the main feature I would install this extension for

**Comment by lvertighel** [comment]
+1 This would make the extension extremely useful on a (capped) mobile connection.

**Comment by InternetCap** [comment]
I signed up to comment on this topic. I have a limited internet data and this would be lifesaver. Im from Turkey and most of my friends have same problem please help us and we will support and donate money thank you and best ragards, Orhan

**Comment by Eloston** [comment]
Okay everyone, here's the current situation.

As you can already see, this issue was marked for the 0.8 release. This is because I will be refactoring the code for version 0.7; the refactoring will cleanup the mess I made thus far and provide a solid base that will make implementing and extending this feature much easier. Now that I have more time and [I have completed changes on another project](https://github.com/Eloston/ungoogled-chromium), I have diverted my efforts on completing version 0.7. You can see my ETA of the version 0.7 release in the [Milestones page](https://github.com/Eloston/disable-html5-autoplay/milestones).

Also, I have noticed that this is the most requested feature for the extension. Because of the high demand, I have decided to get version 0.8 out as soon as possible (I have updated [the milestone](https://github.com/Eloston/disable-html5-autoplay/milestones/0.8) to reflect my new decision). I haven't decided on an ETA for this milestone yet; I will probably set one after I release 0.7.

Thanks for the support everyone.

**Comment by rsyring** [comment]
Thanking for making this a high priority.  I'm another user that now has to get internet through a mobile network and trying to figure out how to limit consumption.  Thanks for making this a priority.

However, I wonder, if the real goal is to conserve bandwidth, if it wouldn't be easier to have an extension dedicated to that.  Something like: limit content downloads if size > X, page downloads if total  bandwidth consumed > x, etc.  I realize out of the scope of this issue, but if you ever decide you wanted to build something like that, I'd love to give some input.  :)

**Comment by AnnoyingMediaBuffs** [comment]
Hello im following this topic and im friend of Orhan From Turkey. I belive you are hope of people like us that technology costs way more than USA or US Or Europa. I hope you will continue being strenght of the weak people, their hope and their hero like Martin Luther King. Best Regards Erhan

**Comment by PJayZ** [comment]
Your extension without stopping the preloading is just useless.

**Comment by HumbleTumble** [comment]
Please ....*groveling* PLEASE!!  Stop the auto buffer.  I was a former  ex-pat in Uganda for the last two years and paid about 60 USD for 10 GB.  I HATED auto--buffering videos!  My in-laws work in Ethiopia where they only get 4GB (when it works) a month.  Opera's Turbo Boost, and user string altering helped a lot - but It would've helped ALOT of people of who have quite little.  (I currently in Opera using the "Download Chrome Extension" extension to).  There were a multitude of sites that I didn't use overseas.  Not because I couldn't, I just couldn't afford the loss of data on something as stupid as a video that I didn't want downloading.  I worked IT at an American school that had a(singular) **TWO** Mb connection - for 400 Students!!  I would have killed for an extension that I could have slapped onto school and staff computers.  I would have sent you postcard greetings and named our firstborn daughter Roberto2525 instead of Natalie in your honor!

I also suggest to you that your user base would increase ...quite a bit possibly, if this feature was marketed as a bandwidth saver.  Additionally, the feature could easily be a toggle, so people who don't care about bandwidth could easily still have the videos buffer in the background.  Remember, people in developing nations have very little internet (Most use less than 1gb per month - probably less) and don't have ANY ability to stop autobuffering.  That feature could be hugely helpful.

**Comment by beggamont** [comment]
for me it would be really great too because i have to come along with one battery charge all day long at university and i have to save ressources everywhere for that
it is really annoying also to pay a lot of moey for mobile data and buffer ads all the time -.-

**Comment by dmayo2** [comment]
+1

Great work on this extension. I too would like buffering to be stopped.

I've noticed that on some sites (can't recall right now) if I want to play the video, I need to turn off the extension and reload the page, as clicking the play button won't work (small inconvenience).

**Comment by bruceleerabbit** [comment]
concur with @PJayZ — this extension is useless to me if it does not stop the buffering.

It’s quite aggravating that in 2023 my limited bandwidth is getting sucked dry by videos I don’t even want on enshitified websites created by reckless scumbags. 

`disable-html5-autoplay` support was dropped in hopes that Google would pick up the ball. But in the end Google’s competency only goes as far as to [mute videos](https://bugs.chromium.org/p/chromium/issues/detail?id=715049). WTF. This even makes Ungoogled Chromium useless to me as I keep getting burnt by costly hit-and-run bandwidth drains.

I must also say the docs are ambiguous. I went through the lengthy manual extension installation process on Ungoogled Chromium only to find that this plugin doesn’t even try to protect bandwidth waste. Whether this request is treated or not, the docs should be clear about what the plugin does in the 1st paragraph.
